tests/other/test_lto_atexit.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+
+// In Wasm destructors get lowered to constructors
+// that call `__cxa_atexit`.
+//
+// Because it gets lowered to this during compilation this
+// symbol cannot itself be LTO.  We has a bug where this symbol
```

Comment:
by the symbols 'being LTO' here, you mean that it's bitcode?

Reply:
Yes.  I see the fact that its bitcode is an implementation detail, but yes.    The limitation is that when a symbol is compiled as LTO we cannot generate new references to it *during* LTO.. because that is too late to fetch it.